src/overview/components/shareButtons.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import styles from './ShareButtons.css'
+
+class ShareButtons extends Component {
+    render() {
+        return (
+            <div className={styles.shareIcons}>
+                <p className={styles.shareText}> Spread the love </p>{' '}
+                <a href="#">
+                    <img
+                        width={25}
+                        height={25}
+                        vspace={2}
+                        src="/img/face.png"
+                    />
+                </a>{' '}
+                <a href="#">
+                    <img
+                        width={25}
+                        height={25}
+                        vspace={2}
+                        src="/img/twitt.png"
+                    />
+                </a>{' '}
+                <a href="#">
```

Comment:
These `<a><img /></a>` element pairs are reused quite often in this markup. May be better to decompose this into their own view that maybe takes in `linkTo` and `imgSrc` as props to clean up this `ShareButtons` view's markup